index.bs

```diff
@@ -593,10 +593,10 @@ Examples for attributes and attribute match lists:
   new Sanitizer({allowAttributes: {"style": ["*"]}}).sanitize(sample);
 
   // Block <span id>: <span class='theclass' style='font-weight: bold'>...</span>
-  new Sanitizer({blockAttributes: {"id": ["span"]}}).sanitize(sample);
+  new Sanitizer({dropAttributes: {"id": ["span"]}}).sanitize(sample);
```

Comment:
The missing braces above looked correct, but why did you cahnge block with drop here?

Reply:
because Spec doesn't has `blockAttributes` so I thought its a miss.
https://wicg.github.io/sanitizer-api/#config

Reply:
oops...fair point :)
Can you change the comment above to also say `drop`?

Reply:
Done, is that works ?

Reply:
Maybe you did not push yet? The code comment still says "block" for me.

Reply:
Oh, my fault. Now its on!